swift-recon.py

```diff
@@ -351,9 +356,19 @@ def main():
             maas_common.status_err('no ring provided to check')
         stats = swift_replication(args.ring)
     else:
-        maas_common.status_err(
-            'unrecognized command "{0}"'.format(args.recon)
-            )
+        raise CommandNotRecognized('unrecognized command "{0}"'.format(
```

Comment:
Yes it can, but this PR is specifically for handling bad subprocess output from swift's recon tool. We can refactor this part in a separate pull request